charts/prometheus-kafka-exporter/README.md

```diff
@@ -4,29 +4,27 @@ A Prometheus exporter for [Apacher Kafka](https://kafka.apache.org/) metrics.
 
 This chart bootstraps a [Kafka Exporter](https://github.com/danielqsj/kafka_exporter) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
 
-
 ## Prerequisites
 
-- Kubernetes 1.8+ with Beta APIs enabled
+- Kubernetes 1.19+
+- Helm 3
+
+Helm v2 was no longer supported from chart version 2.0.0.
```

Comment:
As this is a statement about all the coming chart's releases, "is" would be more appropriate than "was", would it not?

Reply:
fixed.  https://github.com/prometheus-community/helm-charts/pull/3199/commits/a4311e1b542a60ca1cc1364b2f2e295d97d3b82e